src/main/java/org/jpeek/graph/XmlGraph.java

```diff
@@ -66,16 +68,39 @@ public XmlGraph(final Skeleton skeleton) throws IOException {
      * @param skeleton XML representation on whiwh to build the graph
      * @return List of nodes
      * @throws IOException If fails
+     * @todo #408:30min Nodes connections are built based on method calls. For now, the skeleton
+     *  does not reflect which overloaded method is called, so the identification of the called
+     *  method is made by its name. We should wait for #403 to be solved and enhance
+     *  this implementation to consider the overloaded called method.
      */
     private static List<Node> build(final Skeleton skeleton) throws IOException {
-        return new Mapped<XML, Node>(
+        final String clas = skeleton.xml().xpath("//class/@id").get(0);
+        final Map<XML, Node> byxml = new org.cactoos.map.Sticky<>(
+            method -> method,
             method -> new Node.Simple(
                 new Joined(
-                    "", method.xpath("@name").get(0), method.xpath("@desc").get(0)
+                    "", clas, ".", method.xpath("@name").get(0)
```

Comment:
@HDouss You could inline `clas` variable here

Reply:
@paulodamaso Thanks! Fixed.